.github/workflows/remove-labels-on-activity.yml

```diff
@@ -0,0 +1,16 @@
+name: Remove Stale or Waiting Labels
+on:
+  issue_comment:
+    types: [created]
+  workflow_dispatch:
+jobs:
+  remove-labels-on-activity:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions-ecosystem/action-remove-labels@v1
+        if: contains(github.event.issue.labels.*.name, 'Waiting for Response')
+        with:
+          labels: |
+            Waiting for Response
+
```

Comment:
We actually only need this workflow for the "Waiting for Response" label. The stale action will remove the stale label if it sees some activity. I think you actually pointed this out in our last meeting. The only difference is the stale workflow only gets run once a day. The `remove-labels-on-activity` gets run on any comment in the issue.

Reply:
Yeah, but you said it didn't work, then I went back and looked at my own experiment repo and noticed it didn't work either!

Reply:
Yeah I should have explained that a little more in my previous comment. The "Stale" label will only gets removed after the workflow gets ran. So since `stale.yml` only gets ran once a day, there is going to be a little bit of lag time after a new comment is made (possibly a full day worst case). So if we want the label to get removed instantly we could just apply the your changes 🤷‍♂️ either way, I think is fine since the stale label isn't super critical.